server/routes/journeys/record-investigation/protective-factors/view.njk

```diff
@@ -1,41 +1,50 @@
 {% from "govuk/components/character-count/macro.njk" import govukCharacterCount %}
 {% from "govuk/components/details/macro.njk" import govukDetails %}
-{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/inset-text/macro.njk" import govukInsetText %}
+{% from "partials/submitButton/macro.njk" import submitButton %}
 
 {% extends "partials/layout.njk" %}
 
-{% set pageTitle = "Protective factors - Record a CSIP investigation" %}
+{% set pageTitle = "Protective factors - Update a CSIP investigation" if isUpdate else "Protective factors - Record a CSIP investigation" %}
+
 
 {% block innerContent %}
-    <span class="govuk-caption-l">Record a CSIP investigation</span>
+    {% include 'base/components/investigation-caption.njk' %}
     <label for="protectiveFactors">
-        <h1 class="govuk-heading-l govuk-!-margin-bottom-7">What are the protective factors for {{prisoner | firstNameSpaceLastName}}?</h1>
+        <h1 class="govuk-heading-l govuk-!-margin-bottom-7">{{ "Add information about the protective factors for " + (prisoner | firstNameSpaceLastName) if isUpdate else "What are the protective factors for " + (prisoner | firstNameSpaceLastName) + "?" }}</h1>
     </label>
     <form method="post">
         <input type="hidden" name="_csrf" value="{{ csrfToken }}"/>
-        {{ govukDetails({
-            summaryText: "Help with understanding protective factors",
-            html: '
-            <p>Protective factors are anything that helps a prisoner deal more effectively
-              with stressful events or triggers. </p>
-            <p>Include information on any: </p>
-            <ul>
-              <li>positive influences on their behaviour, both historic and recent</li>
-              <li>skills, strengths, resources, supports or coping strategies</li>
-            </ul>'
-        }) }}
+        {% if not isUpdate %}
+            {{ govukDetails({
+                summaryText: "Help with understanding protective factors",
+                html: '
+                <p>Protective factors are anything that helps a prisoner deal more effectively
+                with stressful events or triggers. </p>
+                <p>Include information on any: </p>
+                <ul>
+                <li>positive influences on their behaviour, both historic and recent</li>
+                <li>skills, strengths, resources, supports or coping strategies</li>
+                </ul>'
+            }) }}
+        {% endif %}
+        {% if isUpdate and currentProtectiveFactors %}
```

Comment:
don't need to do both checks here, just checking for the presence of `currentProtectiveFactors` should be enough

Reply:
We do as we shouldnt show the inset text if currentProtectiveFactors is falsey

Reply:
Yeah that's fine, a falsey value for `currentProtectiveFactors` (`undefined`, `null`, or `''`) will fail the if check

Reply:
Done